### DISCUSSION TITLE
Apply Mapbox Studio Styles

### DISCUSSION THREAD

**Discussion by aschenkuttel** [rnmapbox/maps#3184] [Q&A]
Hey,

what is the current proper way of doing this / what should I void.
Simply creating a style via "styleURL" does not work (only a blank map without any text etc)

I'm setting the AccessToken and the Style is published as well.
Could it be due different versions? // V10 here and MapboxStudio maybe already on 11?

Cheers

**Comment by aschenkuttel** [comment]
Ok, pretty sure it's just some wrong attributes due the version difference.
Is there any way of running mapstudio v10 or do I have to manually look after ever style error and disable / change it?

**Comment by denistorjai** [comment]
Did you find a solution?
